pkg/graph/schema.graphql

```diff
@@ -1330,6 +1364,16 @@ type Query {
   systemIntake(id: UUID!): SystemIntake
   systems(after: String, first: Int!): SystemConnection
   currentUser: CurrentUser
+  cedarAuthorityToOperate(
+    cedarSystemID: String,
+    containsPersonallyIdentifiableInformation: Boolean,
+    dispositionDateAfter: Time,
+    dispositionDateBefore: Time,
+    fismaSystemAcronym: String,
+    isProtectedHealthInformation: Boolean,
+    tlcPhase: String,
+    uuid: String
+  ): [CedarAuthorityToOperate!]
```

Comment:
This should be able to return a non-null array (right now the elements are all non-null, but the array itself can be null).

Reply:
Done as of https://github.com/CMSgov/easi-app/pull/1636/commits/b421c6bbac17cdbb5fb9086448d5bcce549eb523.